i18n/de.json

```diff
@@ -9354,10 +9354,6 @@
     "id": "api.templates.delinquency_45.subtitle3",
     "translation": "Aktualisiere jetzt deine Kreditkarteninformationen."
   },
-  {
-    "id": "api.templates.delinquency_45.subtitle2",
-    "translation": "Ein herabgestufter Arbeitsbereich kann sich negativ auf kritische Arbeitsabläufe, Integrationen und andere geschäftskritische Aktivitäten in deinem Arbeitsbereich auswirken."
-  },
```

Comment:
Removing these translations for them to be re-translated seems to be the best bet, as the change in wording might affect the layout/structure of the text and isn't as simple as updating a name.